services/mgmt/compute/examples/vm_list_using_resource_graph.rs

```diff
@@ -0,0 +1,85 @@
+use std::sync::Arc;
+use azure_identity::DefaultAzureCredential;
+use azure_mgmt_resourcegraph::models::{QueryRequest, QueryRequestOptions};
+use serde_json::{json, Value};
+use azure_mgmt_resourcegraph::Client as AzureResourceGraphClient;
+use azure_mgmt_resourcegraph::ClientBuilder as AzureResourceGraphClientBuilder;
+
+/// Fetching Virtual Machines using Azure Resource Graph
+
+#[derive(Debug)]
+pub enum GenericError {
+    InvalidRequest,
+}
+
+#[derive(Debug)]
+struct CustomError {
+    err_type: GenericError,
+    err_msg: String
+}
+
+async fn fetch_azure_records() -> Result<Value, CustomError> {
+
+    let azure_creds = Arc::new(DefaultAzureCredential::default());
+
+    let az_rg_client = AzureResourceGraphClient::builder(azure_creds).build();
```

Comment:
```suggestion
    let az_rg_client = Client::builder(azure_creds).build();
```

Reply:
I actually had multiple Clients in my code, and not just AzureResourceGraphClient. Hence, explicitly used it. But this is just one example, we can use Client as is.